crates/telio-model/src/api_config.rs

```diff
@@ -164,7 +164,7 @@ pub struct FeatureDirect {
     /// Endpoint providers [default all]
     #[serde(default)]
     #[serde(deserialize_with = "deserialize_providers")]
-    pub providers: Option<HashSet<EndpointProvider>>,
+    pub providers: Option<Vec<EndpointProvider>>,
```

Comment:
UniFFI can't deal with hash sets so replaced with vector

Reply:
I tried using a type alias for `HashSet<EndpointProvider>` and I tried creating a wrapper, and neither worked. I think it should be possible to have a wrapper but so far I feel like it's much more work than it's worth given how little we actually use the endpoint providers. Instead I added deduplication of endpoint providers to the constructor for `Device`

Reply:
I managed to fix it with a type alias